File: Is Even/isEven.hs
isEven :: Integer -> Bool
isEven = (==0) . (`mod` 2)

main :: IO()
main = do
    putStrLn "Enter the number: "
    num <- getLine
    if (isEven $ read num) then
        putStrLn("The number is even.")
    else
        putStrLn("The number is odd.")

File: Is Even/isEven.swift
import Foundation

func checkEvenOrOdd(number: Int) -> String {
    if number % 2 == 0 {
        return "The number \(number) is even."
    } else {
        return "The number \(number) is odd."
    }
}

let number = 7
let result = checkEvenOrOdd(number: number)
print(result)
